exec/src/coqnum.ml

```diff
@@ -49,6 +62,7 @@ let int_of_coqz z =
         | _ -> raise (UnsupportedTerm "not a constructor of [Z]"))
   | _ -> raise (UnsupportedTerm "not a constructor of [Z]")
 
-
-let int_to_coqz _ =
-  raise (UnsupportedTerm "To be implemented")
+let int_to_coqz = function
+  | x when x > 0 -> mkApp (Ind.Z.mkConstructor "Zpos", Array.of_list [int_to_coqpositive x])
+  | 0 -> Ind.Z.mkConstructor "Z0"
+  | x -> mkApp (Ind.Z.mkConstructor "Zneg", Array.of_list [int_to_coqpositive (-1 * x)])
```

Comment:
I wonder if using the `abs` function from Pervasive wouldn’t make the idea behind `-1 * x`?